src/shogun/features/DenseFeatures.cpp

```diff
@@ -170,28 +170,6 @@ template<class ST> ST* CDenseFeatures<ST>::get_feature_vector(int32_t num, int32
 	return feat;
 }
 
-template<class ST> void CDenseFeatures<ST>::set_feature_vector(SGVector<ST> vector, int32_t num)
```

Comment:
it is not used?

Reply:
yes, and i think `set_feature_matrix` should be also removed (after some refactor)

Reply:
absolutely, the instance should be tied to the matrix, and otherwise users can create a new instance